code/cfile/cfilesystem.cpp

```diff
@@ -1744,6 +1747,9 @@ int cf_get_file_list(SCP_vector<SCP_string>& list, int pathtype, const char* fil
 	}
 
 
+    // tcrayford: sort the filesystem listing by name by default to ensure
+    // a stable order across filesystems
+    cf_sort_filenames( list, CF_SORT_NAME, info );
```

Comment:
Please keep the style consistent in the file (indent using tabs).

Reply:
Fixed.